vip-datamanagement/src/main/java/fr/insalyon/creatis/vip/datamanager/client/bean/ExternalPlatform.java

```diff
@@ -46,6 +46,9 @@ public enum Type {
     private Type type;
     private String description;
     private String url;
+    private String upload_url;
+    private String keycloak_client_id;
+    private String refresh_token_url;
```

Comment:
Please use camel case for fields names : `uploadUrl`

Reply:
Done, you can check the changes.